www/src/utils/findIndexOf/findIndexOf.test.ts

```diff
@@ -0,0 +1,43 @@
+import findIndexOf from './findIndexOf';
```

Comment:
Nice for creating utils.

Can we please change it to the following:

```
|-- utils
     |-- tests
           |-- stringUtils.tests.ts
     |-- stringUtils.ts
```

Also please remove the index.ts file as it does not support tree shaking.

Reply:
sure thing :)